server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/NodeType.java

```diff
@@ -49,5 +49,6 @@
   COPY,
   DIVISION,
   LITERAL,
-  COPY_DEFINITION_NODE
+  SUBROUTINE,
+  SUBROUTINE_USAGE_NODE
```

Comment:
The names of the types are confusing, SUBROUTINE_USAGE_NODE is used for the names of the subroutines, if I understood correctly

Reply:
Does make sense. Updated